public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -117,25 +148,61 @@ class Findings extends Component<FindingsProps, FindingsState> {
     const timeUnits = getChartTimeUnit(dateTimeFilter.startTime, dateTimeFilter.endTime);
     this.state = {
       loading: true,
-      findings: [],
       notificationChannels: [],
-      rules: {},
+      selectedTabId: FindingTabId.DetectionRules,
+      findingStateByTabId: {
+        [FindingTabId.DetectionRules]: {
+          findings: [],
+          rules: {},
+          filteredFindings: [],
+          groupBy: 'logType',
+        },
+        [FindingTabId.ThreatIntel]: {
+          findings: [],
+          filteredFindings: [],
+          groupBy: 'indicatorType',
+        },
+      },
       recentlyUsedRanges: [DEFAULT_DATE_RANGE],
-      groupBy: 'logType',
-      filteredFindings: [],
-      plugins: [],
       timeUnit: timeUnits.timeUnit,
       dateFormat: timeUnits.dateFormat,
     };
   }
 
+  shouldUpdateVisualization(prevState: FindingsState) {
```

Comment:
rename `shouldUpdateVisualizationState` ?

Reply:
It is actually checking if the visualization should be updated, so keeping it as is